License URL is wrong in pom file

I am using the license-maven-plugin and when i am downloading the licenses the plugin fails with

[INFO] --- license:2.4.0:download-licenses (default-cli) @ efnol ---
ERROR StatusLogger Log4j2 could not find a logging implementation. Please add log4j-core to the classpath. Using SimpleLogger to log to the console...
[ERROR] 'https://github.com/loki4j/loki-logback-appender/blob/master/LICENSE.md' returned 404 Not Found 

As the url to the license.md is just pointing to the wrong place

https://github.com/loki4j/loki-logback-appender/blob/master/LICENSE.md this points to a 404 page
But it actually should point to: https://github.com/loki4j/loki-logback-appender/blob/main/LICENSE

--- a/gradle/publishing.gradle
+++ b/gradle/publishing.gradle
@@ -8,7 +8,7 @@ plugins.withType(MavenPublishPlugin) {
                     licenses {
                         license {
                             name = 'BSD Software License, 2-clause version'
-                            url = 'https://github.com/loki4j/loki-logback-appender/blob/master/LICENSE.md'
+                            url = 'https://github.com/loki4j/loki-logback-appender/blob/master/LICENSE'
                         }
                     }
                     developers {

